recipes/heavydb/build-heavydb.sh

```diff
@@ -0,0 +1,327 @@
+        else
+            export RUN_TESTS=0
+        fi
+    fi
+else
+    export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DENABLE_CUDA=off"
+    if [[ "$RUN_TESTS" == "2" ]]
+    then
+        export RUN_TESTS=1
+    fi
+fi
+
+if [[ "$RUN_TESTS" == "0" ]]
+then
+   export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DENABLE_TESTS=off"
+else
+   export RUN_TESTS=1
+   export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DENABLE_TESTS=on"
+fi
+
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DBoost_NO_BOOST_CMAKE=on"
+
+# As clang is used in the build process to compile bytecode,
+# we need to make clang aware of the C++ headers provided by
+# conda's GCC toolchain.
+. ${RECIPE_DIR}/get_cxx_include_path.sh
+export CPLUS_INCLUDE_PATH=$(get_cxx_include_path)
+
+mkdir -p build
+cd build
+
+# Run configure
+case "$PKG_NAME" in
+
+    heavydb | heavydbe | pyheavydbe | heavydb-common)
+
+        cmake -Wno-dev \
+              -DCMAKE_PREFIX_PATH=$PREFIX \
+              -DCMAKE_INSTALL_PREFIX=$PREFIX/$INSTALL_BASE \
+              -DCMAKE_BUILD_TYPE=release \
+              -DMAPD_DOCS_DOWNLOAD=off \
+              -DENABLE_AWS_S3=off \
+              -DENABLE_FOLLY=off \
+              -DENABLE_JAVA_REMOTE_DEBUG=off \
+              -DENABLE_PROFILER=off \
+              -DPREFER_STATIC_LIBS=off \
+              $EXTRA_CMAKE_OPTIONS \
+              ..
+
+        ;;
+
+    *)
+        echo "Nothing configured for $PKG_NAME"
+        ;;
+
+esac
+
+# Run build
+case "$PKG_NAME" in
+
+    heavydb-common)
+        echo "Building heavydb-common"
+        ninja QueryEngineFunctionsTargets mapd_java_components generate_cert_target
+        ;;
+
+    heavydbe)
+        echo "Building heavydbe"
+        ninja DBEngine
+        ;;
+
+    pyheavydbe)
+        echo "Installing pyheavydbe"
+        cd Embedded
+        $PYTHON -m pip install .
+        cd ..
+        ;;
+
+    heavydb)
+        echo "Building heavydb"
+        if [[ "$BUILD_EXT" == "cuda" ]]
+        then
+            ninja initheavy heavydb heavysql StreamImporter KafkaImporter QueryEngine/cuda_mapd_rt.fatbin
+        else
+            ninja initheavy heavydb heavysql StreamImporter KafkaImporter
+        fi
+        ;;
+
+    *)
+        echo "Nothing build for $PKG_NAME"
+        ;;
+
+esac
+cd ..
+
+# Run install
+case "$PKG_NAME" in
+
+    heavydb-common)
+        echo "Installing heavydb-common"
+        cmake --install build --component "include" --prefix $PREFIX/$INSTALL_BASE
+        cmake --install build --component "doc" --prefix $PREFIX/share/doc/heavyai
+        cmake --install build --component "data" --prefix $PREFIX/$INSTALL_BASE
+        cmake --install build --component "thrift" --prefix $PREFIX/$INSTALL_BASE
+        cmake --install build --component "QE" --prefix $PREFIX/$INSTALL_BASE
+        cmake --install build --component "jar" --prefix $PREFIX/$INSTALL_BASE
+        cmake --install build --component "Unspecified" --prefix $PREFIX/$INSTALL_BASE
```

Comment:
```suggestion
        cmake --install build --component "include" --prefix $PREFIX
        cmake --install build --component "doc" --prefix $PREFIX/share/doc/heavyai
        cmake --install build --component "data" --prefix $PREFIX
        cmake --install build --component "thrift" --prefix $PREFIX
        cmake --install build --component "QE" --prefix $PREFIX
        cmake --install build --component "jar" --prefix $PREFIX
        cmake --install build --component "Unspecified" --prefix $PREFIX
```

Would this work out of the box?